client/components/mma/identity/MarketingCheckbox.tsx

```diff
@@ -38,18 +38,11 @@ const getDescription = (description: MarketingCheckboxProps['description']) => (
 
 export const MarketingCheckbox: FC<MarketingCheckboxProps> = (props) => {
 	const { id, description, selected, title, onClick } = props;
+
 	return (
 		<div
 			data-cy={id}
-			onClick={(e) => {
-				// Checkboxes inside labels will trigger click events twice.
-				// Ignore the input click event
-				if (
-					e.target instanceof Element &&
-					e.target.nodeName === 'INPUT'
-				) {
-					return;
-				}
```

Comment:
This used to be needed in order to catch the onClick event of the hidden checkbox. Not needed now that the hidden checkbox is gone.